src/julia_internal.h

```diff
@@ -504,3 +504,6 @@ STATIC_INLINE void jl_free_aligned(void *p)
 #endif
 
 #endif
+
+#define SMALL_BYTE_ALIGNMENT 16
```

Comment:
even though this is the internal header, still probably best to prefix them with `JL_`

these should also be inside the include guard